### ISSUE TITLE
Can we send messages using bailyes to whatsapp channel ??

### ISSUE THREAD

**Issue by vihangayt0** [WhiskeySockets/Baileys#594]
Can we send messages using bailyes to whatsapp channel ??

**Comment by vihangayt0** [comment]
@PurpShell @Auties00 pls help me

**Comment by chocolaid** [comment]
Yes you can send message to WhatsApp channel use the channel id as the chat id

**Comment by vihangayt0** [comment]
> Yes you can send message to WhatsApp channel use the channel id as the chat id

but not work

**Comment by chocolaid** [comment]
Let me see your current code
